api/workload/v2/backoff.go

```diff
@@ -0,0 +1,34 @@
+package workload
+
+import (
+	"math"
+	"time"
+)
+
+// backoff defines an linear backoff policy.
+type backoff struct {
```

Comment:
Would it make sense to move this to some common location? This seems like it is very generalized and may be useful elsewhere.

Reply:
That's a good idea. I think we can submit this as is and do that refactor once we have the need for it.